<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>screen3</class>
 <widget class="QDialog" name="screen3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FRAIL2FIT</string>
  </property>
  <widget class="QLabel" name="q1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>450</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;What advice would you give to someone who is just starting their fitness journey ?&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="q1option1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>450</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Being consistent</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="q1option2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>450</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Following the diet properly</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="q1option3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>450</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Listen to your body</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="q1option4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>450</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Set realistic goals</string>
   </property>
  </widget>
  <widget class="QLabel" name="q2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>300</y>
     <width>450</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>2. How often do you workout in a week?</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="q2option1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>340</y>
     <width>450</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Never but I want to start</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="q2option2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>380</y>
     <width>450</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>once per week</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="q2option3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>420</y>
     <width>450</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>2-3 days a week</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="q2option4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>460</y>
     <width>450</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>4+ days a week</string>
   </property>
  </widget>
  <widget class="QLabel" name="q3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>520</y>
     <width>450</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Did you ever workout till date?&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="s3nxt">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>710</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="yesButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>570</y>
     <width>460</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Yes</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="noButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>610</y>
     <width>460</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>No</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
